RDBM/XML.h

```diff
@@ -0,0 +1,285 @@
+#ifndef XML_h_
+#define XML_h_
+#include"Debug\tinyxml.h"
+#include"Table.h"
+
+void XMLDecl()
+{
+	TiXmlDocument doc;
+	doc.LoadFile("dbdata.xml");
+	TiXmlDeclaration *decl = new TiXmlDeclaration("1.0", "UTF-8", "");
+	doc.LinkEndChild(decl);
+	TiXmlElement *list = new TiXmlElement("List");
+	doc.LinkEndChild(list);
+	doc.SaveFile("dbdata.xml");
+
+	TiXmlDocument doc2;
+	doc2.LoadFile("dbstructure.xml");
+	TiXmlDeclaration *decl2 = new TiXmlDeclaration("1.0", "UTF-8", "");
+	doc2.LinkEndChild(decl2);
+	TiXmlElement *list2 = new TiXmlElement("List");
+	doc2.LinkEndChild(list2);
+	doc2.SaveFile("dbstructure.xml");
+}
+
+bool check(Table& tb)
+{
```

Comment:
I don't understand.
This function returns false if current table was found in xml and true if not?

Reply:
yes.if table was found it means that it must be overwritten.

Reply:
Please, rename current function to sth like this "NeedsToBeOverwritten" and return true if such table would be found and false in the other case.

Reply:
as you wish